CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [0.14.0] - 2021-08-09
```

Comment:
We'll need to update this date when we actually release the change (probably tomorrow). We would generally leave the date here as `Unreleased` until then but it's not a huge deal.

Also, if you check the bottom of this file you'll see a bunch of lines that look like `[version_number]: link_to_github_diff`. Mind adding one for this change entry? That's the markdown that'll turn the [0.14.0] into a hyperlink.